tasks/configure_user.yaml

```diff
@@ -0,0 +1,41 @@
+---
+- name: "check for required fact - name"
+  fail:
+    msg: "missing required fact: name"
+  with_items: "{{ users }}"
+  when: ( user.name is not defined )
+  loop_control:
+    loop_var: user
+  delegate_to: localhost
+
+- name: "display input"
+  debug:
+    var: users
+  delegate_to: localhost
```

Comment:
is this debug out required ? anyway one can see stdout using -vv option in first task.

Reply:
+1. I too wasn't entirely sure about this.